Pipfile

```diff
@@ -0,0 +1,11 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+
+[requires]
+python_version = "3.8"
```

Comment:
Looks like `pipenv install` was ran on the front-end repo. 
This file can be deleted - It does not belong on the FE repo.

Reply:
Removed that file